src/ios.rs

```diff
@@ -1,5 +1,7 @@
 #![cfg(target_os = "ios")]
 
+const EPSILON: f64 = 0.0001;
```

Comment:
https://doc.rust-lang.org/std/primitive.f64.html#associatedconstant.EPSILON

Reply:
I think in this case the epsilon is more about slight differences in the two time APIs than necessarily a floating point rounding issue, so the machine epsilon value is a bit too low. The assert hit I was getting was in this scenario:

```
start_os: 115413.6942321667, os_cur_time: 115413.69124966669
thread '<unnamed>' panicked at 'assertion failed: start_os <= os_cur_time', /Users/arthurvalladares/Documents/GitHub/display-link/src/ios.rs:128:17
```

Reply:
Gotcha, thanks!